pkg/profileparser/profileparser.go

```diff
@@ -621,10 +638,11 @@ func ParseRulesAndDo(contentDom *xmlquery.Node, stdParser *referenceParser, pb *
 				},
 			}
 			if description != nil {
-				p.Description = utils.XmlNodeAsMarkdown(description)
+				p.Description, _ = utils.RenderValues(utils.XmlNodeAsMarkdownPreRender(description), valuesList)
```

Comment:
I agree with not passing an error up, but I think we could at least log the error and not ignore it.